front/lib/api/wrappers.ts

```diff
@@ -254,3 +295,61 @@ export function withPublicAPIAuthentication<T, U extends boolean>(
     isStreaming
   );
 }
+
+/**
+ * This function is a wrapper for Public API routes that require authentication without a workspace with a token
+ */
+export function withUserTokenAuthentication<T>(
```

Comment:
Why UserToken here when it’s is Auth0Token everywhere else? Should we standardize on one name?